report/main.qmd

```diff
@@ -1 +1,166 @@
+         filter(Intervention_new!="wash" & Intervention_new !="wash2"), 
+       aes(x=Intervention_new, y=ZeroCal_percent)) +
+  geom_boxplot() + 
+  facet_wrap( ~ Site, ncol = 3, scales = "free_y") + 
+  xlab("Intervention") + 
+  ylab("Daily percentage of \nzero-calorie beverages sales (%)") +
+  ggtitle("",subtitle = "Figure 3: Boxplots of daily percentage of zero-calorie beverages sales \nfor each of the three sites") + 
+  theme(axis.text.x = element_text(angle = 90, vjust = 0.5, hjust=1))
+
+```
+
+
+## Formal Analysis
+To see which interventions lead to increases in zero-calorie beverages consumption and/or decreases in sugary beverages consumption, it is recommended to use generalized least squares regression (GLS). GLS is similar to linear regression. The key difference is that GLS allows the residuals to be correlated while linear regression assumes that the residuals are independent. For time series data, as the one that we have, observations close to one another may be more likely to be similar than observations that are relatively remote. Therefore, it may not be reasonable to assume that the residuals of a regression model are independent, and so GLS is preferred over linear regression.
+
+It is recommended to fit GLS models separately for each of the three sites, with zero-calorie beverages percentage or sugary beverages percentage as the response variable. In addition to the reason mentioned in the second paragraph of section 3, another advantage of using percentage terms over raw sales is that we just need to fit models for either zero-calorie beverages percentage or sugary beverages percentage instead of for both. An increase in zero-calorie beverages percentage implies a decrease in sugary beverages percentage, vice versa.
+
+Regarding the predictor variables,  we recommend using `Intervention` and `DofW`. For `Intervention`, it is important to first rename some of the categories. For the observations made in the second pre-intervention period and the second follow-up period, the values of `Intervention` should be respectively replaced by something like "preint2" and "follow2". This way we are not mixing up the first and second pre-intervention and follow-up period in our analysis. Also, "wash" and "wash2" can be combined into one category as they are not of the main interest in the study. For `DofW`, Monday through Friday can be merged into a category called weekday, and Saturday and Sunday can be merged into a category called weekend. When adding text label of `DofW` to each of the data points in Figure 2, it can be seen that most of the valleys of the time series happen at weekends. Therefore, it is important to control for day of week when making comparisons between intervention and pre-intervention periods. In summary, the non-residual part of each site-specific model can be specified as ZeroCal_percent ~ Intervention_updated + DofW_updated or Sugary_percent ~ Intervention_updated + DofW_updated.
```

Comment:
Don't write the `R` formula code into your paragraph like that. Math is math, text should read like text, and code shouldn't appear in your report.